starwarsapp/src/components/search/SearchResult.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <div class="search-result flex-h align-center justify-space-between mt-m">
+    <span class="search-result-name">{{ name }}</span>
+    <base-button class="go-to-details px-m" text="Details"/>
+  </div>
+</template>
+
+<script>
+import BaseButton from '../common/BaseButton'
+
+export default {
+  name: 'search-result',
+  props: {
+    name: {
+      type: String,
+      required: true
+    },
+    link: {
+      type: String,
+      required: true
+    }
+  },
+  computed: {
+    url () {
```

Comment:
Change the computed name to something more indicitive of what the computed variable is returning